mirrord/cli/src/internal_proxy.rs

```diff
@@ -159,6 +160,9 @@ pub(crate) async fn proxy(args: InternalProxyArgs) -> Result<()> {
         })??;
     }
 
+    let (main_connection_cancalation_token, main_connection_task_join) =
```

Comment:
```suggestion
    let (main_connection_cancellation_token, main_connection_task_join) =
```
Plus more like this below :p